pkg/github/workflows_handler.go

```diff
@@ -39,3 +39,19 @@ func (s *QueryHandler) HandleWorkflowUsage(ctx context.Context, req *backend.Que
 		Responses: processQueries(ctx, req, s.handleWorkflowUsageQuery),
 	}, nil
 }
+
+func (s *QueryHandler) handleWorkflowRunsQuery(ctx context.Context, q backend.DataQuery) backend.DataResponse {
+	query := &models.WorkflowRunsQuery{}
+	if err := UnmarshalQuery(q.JSON, query); err != nil {
+		return *err
```

Comment:
Should this be marked as downstream error?

Reply:
this follows the same pattern as the other query handlers, so i think its okay as is.

though, i don't have much experience with err source, if you think this is something we should change in all handler methods, we can continue to discuss but its out of the scope of this pr.

example: https://github.com/grafana/github-datasource/blob/a9f2d33e0c0042faa42f82de94b7931c119a2435/pkg/github/commits_handler.go#L11-L17